torch/csrc/jit/passes/onnx/helper.h

```diff
@@ -19,22 +23,29 @@ using ValueToParamPairMap = std::map<Value*, std::pair<std::string, IValue>>;
 
 using ParamMap = std::map<std::string, IValue>;
 
-void buildParamsMapFromValueToParamsMap(
+TORCH_API void buildParamsMapFromValueToParamsMap(
     const ValueToParamPairMap& valsToParamsMap,
     ParamMap& paramsDict);
-ValueToParamPairMap buildValueToParamsMap(Block* b, const ParamMap& paramsDict);
-void eraseUnusedValuesFromMap(ValueToParamPairMap& valsToParamsMap);
-void eraseUnusedBlockInputs(Block* b);
-void buildParamsMapFromValueToParamsMap(
+TORCH_API ValueToParamPairMap
+buildValueToParamsMap(Block* b, const ParamMap& paramsDict);
+TORCH_API void eraseUnusedValuesFromMap(ValueToParamPairMap& valsToParamsMap);
+TORCH_API void eraseUnusedBlockInputs(Block* b);
+TORCH_API void buildParamsMapFromValueToParamsMap(
     const ValueToParamPairMap& valsToParamsMap,
     ParamMap& paramsDict);
 
-Node* addNodeToBlock(Block* block, Symbol kind, ArrayRef<Value*> inputs);
+TORCH_API Node* addNodeToBlock(
+    Block* block,
+    Symbol kind,
+    ArrayRef<Value*> inputs);
 
-Value* addInputToBlock(Block* block);
+TORCH_API Value* addInputToBlock(Block* block);
 
 TORCH_API c10::optional<at::ScalarType> ONNXTypeToATenType(int32_t onnx_type);
 
+TORCH_API ONNX_NAMESPACE::TensorProto_DataType ATenTypeToOnnxType(
```

Comment:
There are also  a `torch:onnx` namespace, which does not define `TensorProto_DataType`, does it?
```suggestion
TORCH_API ::ONNX_NAMESPACE::TensorProto_DataType ATenTypeToOnnxType(
```